app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -163,15 +163,14 @@ def handle_bad_request_error(err, enrollment)
 
     if response_message == IPP_INCOMPLETE_ERROR_MESSAGE
       # Customer has not been to post office for IPP
```

Comment:
can we update this comment as now we can mark someone incomplete for the additional reason of blocking expirations

Reply:
well actually i guess we dont need to since it's still true that the customer hasn't gone into the po. So ignore the previous comment